### ISSUE TITLE
5.0.0 running failed

### ISSUE THREAD

**Issue by lcgogo** [input-output-hk/cardano-graphql#572]
### Environment
The [CLI](../../packages/cli) can be used to gather generic information about your system to assist with support.
No installation is required thanks to [npx](https://github.com/npm/npx)

``` console
npx @cardano-graphql/cli system-info
```
[PASTE_REPORT_HERE]

**Alternatively, what release or Git revision are you using:**
5.0.0

```shell
Installing   cardano-crypto-wrapper-1.3.0 (lib)
Completed    cardano-crypto-wrapper-1.3.0 (lib)
Installing   iohk-monitoring-0.1.10.1 (lib)
Completed    iohk-monitoring-0.1.10.1 (lib)
Installing   statistics-0.15.2.0 (lib)
Completed    statistics-0.15.2.0 (lib)

Failed to build cardano-crypto-class-2.0.0.
Build log (
/root/.cabal/logs/ghc-8.10.4/cardano-crypto-class-2.0.0-429390cef1dc0c622521c8f5c8c69a6eff875ad03a076199192658fc6
e841bd3.log
):
Configuring library for cardano-crypto-class-2.0.0..
Preprocessing library for cardano-crypto-class-2.0.0..
compiling dist/build/Cardano/Crypto/Libsodium/Constants_hsc_make.c failed (exit code 1)
rsp file was: "dist/build/Cardano/Crypto/Libsodium/hsc2hscall8652-0.rsp"
command was: /usr/bin/cc -c dist/build/Cardano/Crypto/Libsodium/Constants_hsc_make.c -o dist/build/Cardano/Crypto
/Libsodium/Constants_hsc_make.o -fuse-ld=gold -D__GLASGOW_HASKELL__=810 -Dlinux_BUILD_OS=1 -Dx86_64_BUILD_ARCH=1
-Dlinux_HOST_OS=1 -Dx86_64_HOST_ARCH=1 -I/usr/local/include -I/usr/local/include -Idist/build/autogen -Idist/buil
d/global-autogen -include dist/build/autogen/cabal_macros.h -I/root/.cabal/store/ghc-8.10.4/vector-0.12.3.0-faf76
43b44b1f151021734a55226cb753a5aa672fe437a88c8d4789bace1ff48/lib//include -I/opt/ghc/8.10.4/lib/ghc-8.10.4/unix-2.
7.2.2/include -I/opt/ghc/8.10.4/lib/ghc-8.10.4/time-1.9.3/include -I/root/.cabal/store/ghc-8.10.4/primitive-0.7.1
.0-57016f0038ed9bc68ae4ba8a0bf5334da1a65f93f4331980931e581cdbba846c/lib//include -I/opt/ghc/8.10.4/lib/ghc-8.10.4
/bytestring-0.10.12.0/include -I/opt/ghc/8.10.4/lib/ghc-8.10.4/base-4.14.1.0/include -I/opt/ghc/8.10.4/lib/ghc-8.
10.4/integer-gmp-1.0.3.0/include -I/opt/ghc/8.10.4/lib/ghc-8.10.4/include -I/opt/ghc/8.10.4/lib/ghc-8.10.4/includ
e/
error: In file included from Constants.hsc:15:
/usr/local/include/sodium.h:50:10: fatal error: sodium/crypto_vrf.h: No such file or directory
 #include "sodium/crypto_vrf.h"
          ^~~~~~~~~~~~~~~~~~~~~
compilation terminated.

cabal: Failed to build cardano-crypto-class-2.0.0 (which is required by
exe:cardano-node from cardano-node-1.28.0). See the build log above for
details.

ERROR: Service 'cardano-node-ogmios' failed to build : The command '/bin/sh -c cabal install cardano-node   --ov$
rwrite-policy=always   --install-method=copy   --installdir=/app/bin' returned a non-zero code: 1
```

#### Platform

- [x] Linux / Ubuntu
- [ ] Linux / Other
- [ ] macOS
- [ ] Windows

**Platform version**: 18.04

#### Runtime

- [ ] Node.js
- [x] Docker

**Runtime version**: 
```console
node -v
docker version
```

### What is the expected behavior?

**Comment by rhyslbw** [comment]
Did you follow the [getting started instructions](https://github.com/input-output-hk/cardano-graphql/tree/5-0-stable#getting-started)?

**Comment by felipe2g** [comment]
> Did you follow the [getting started instructions](https://github.com/input-output-hk/cardano-graphql/tree/5-0-stable#getting-started)?

Hello! I'm getting the same in MacOS. I followed all starter instructions.

**Comment by longngn** [comment]
I'm getting the same error

**Comment by rhyslbw** [comment]
Ack. This is caused by a "stable" dependency being dropped:

https://github.com/CardanoSolutions/ogmios/blob/8bd814d8c47eceeb30949185a36164a94f83956c/Dockerfile#L10
https://github.com/input-output-hk/libsodium/search?q=66f017f16633f2060db25e17c170c2afa0f2a8a1

We're coordinating with the maintainers to understand why this happened, and will advise ASAP.

**Comment by rhyslbw** [comment]
Looking into this now

**Comment by rhyslbw** [comment]
This has been resolved in [5.1.0-beta.1](https://github.com/input-output-hk/cardano-graphql/releases/tag/5.1.0-beta.1), so depending on your use-case upgrading could be the quickest way to get unstuck. If deploying to a production environment where building `5.0.0` is required, moving [this line](https://github.com/CardanoSolutions/ogmios/blob/a2a92ee54b08e9b8b3c8cc95ea4f4c4d83904579/Dockerfile#L7) in the Ogmios Dockerfile to line 14 will  fix the issue

```
#                                                                              #
# ------------------------------- SETUP  ------------------------------------- #
#                                                                              #

FROM haskell:8.10.4 as setup
ARG IOHK_LIBSODIUM_GIT_REV=66f017f16633f2060db25e17c170c2afa0f2a8a1

```
